### ISSUE TITLE
multiple huffman tree

### ISSUE THREAD

**Issue by da0ka** [kapenga/LittleBit#5]
How about use 2 huffman trees? One used for dictionary, and for packed data. Of course, it hurts speed.

**Comment by kapenga** [comment]
I don't think it would help compression. Storing a tree gives probably more overhead than gaining on the nodes.
I however made some uncommitted changes to the tree generation to avoid leaves that are only used for the tree itself. The result is that the tree is slightly smaller and compression itself improves too.
In the future I will commit these changes to github.
